capsul/application.py

```diff
@@ -40,17 +41,13 @@ class Capsul:
 
     '''    
 
-    def __init__(self, config_file=None):
-        if config_file is None:
-            self.config_file = None
-            self.config = {}
-        else:
-            if isinstance(config_file, str):
-                self.config_file = Path(config_file)
-            else:
-                self.config_file = config_file
-            with self.config_file.open() as f:
-                self.config = json.load(f)
+    def __init__(self, config_file=None, site_config_file=None):
+        self.config = ApplicationConfiguration('capsul', user_file=config_file,
```

Comment:
The name of the application is `capsul` harcoded. It should probably be a parameter of `Capsul` constructor (why not with a `capsul` default value).

Reply:
Sure, it was set temporarily. I'm changing that